app/controllers/triage_controller.rb

```diff
@@ -4,7 +4,7 @@ class TriageController < ApplicationController
   before_action :set_triage, only: %i[show]
   before_action :set_consent_response, only: %i[show]
 
-  layout "two_thirds", except: %i[index]
```

Comment:
@theo explicitly said the index page shouldn't be in this layout. I don't recall the reasoning why though.

Ship it and sort it out later?

Reply:
I'm changing the layout of the index immediately after this PR, so this is deliberate.

The new designs make the table narrower to leave room for the filters in future:

<img width="1007" alt="image" src="https://github.com/nhsuk/record-childrens-vaccinations/assets/23801/85b51f32-744f-4897-be78-a021a34353a6">